pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -52,7 +52,7 @@ The supported labels are:
 
 * `title`: The policy name that appears in the UI.
 * `description`: A brief explanation of the policy's behavior.
-* `category`: The category the policy appears under. Supported values are: `security`, `compliance`, `transformation`, `quality of service`, and `troubleshooting`.
+* `category`: The category the of the policy. Suggested categories are `security`, `compliance`, `transformation`, `quality of service`, and `troubleshooting`.
```

Comment:
```suggestion
* `category`: The category the of the policy. The possible categories are `security`, `compliance`, `transformation`, `quality of service`, and `troubleshooting`.
```
"Suggested" implies that there are other possible categories. Is this the intent? If so then please disregard.